### PR TITLE
add CODEOWNERS

### PR DESCRIPTION
**Pull request by jmartin-tech**
Enforce approval requirements for changes that may impact automation.

### CODE REVIEW

**Comment by jmartin-tech** [comment]
See [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+/.github/ @leondz @erickgalinkin @jmartin-tech
```
[the remark this answers is not in the document]
Comment (jmartin-tech) [reply .github/CODEOWNERS:1]: I would say no, I do not currently want all PRs to auto-request review.

This is intended for protection of automation artifacts by requiring a maintainer to approve landing any PR that may change those workflows. I can see a future where there are experts on certain segments of the codebase that would be desirable to tag on reviews of changes in those areas, the project is not that complex at this time.

[the remark this answers is not in the document]
Comment (leondz) [reply .github/CODEOWNERS:1]: I think having a `.github` restriction, separate from global config/restrictions, is a good extra layer of protection

**Review by leondz — APPROVED** [review]
nice

**Pull request merged**